docs/CHANGELOG.rst

```diff
@@ -47,6 +48,7 @@ Fixed
 - Data objects created by the workflow inherite it's permissions
 - If user doesn't have permissions on the latest versions of processes
   and descriptor schemas, use older ones except returning error
+- Support data: and list:data types
```

Comment:
Usually we omit the last `:`, because our platform handles it automatically. Since `data` was uninformative I decided to make an exception here and write `data:`.